bindings/python/capstone/__init__.py

```diff
@@ -708,6 +726,8 @@ def __gen_detail(self):
             (self.operands) = bpf.get_arch_info(self._raw.detail.contents.arch.bpf)
         elif arch == CS_ARCH_RISCV:
             (self.operands) = riscv.get_arch_info(self._raw.detail.contents.arch.riscv)
+        elif arch == CS_ARCH_TRICORE:
+            (self.operands) = riscv.get_arch_info(self._raw.detail.contents.arch.tricore)
```

Comment:
Should this be `tricore.get_arch_info` instead of `riscv.get_arch_info`?

Reply:
You are right

Reply:
@peace-maker Could you please send a new PR to fix these errors?